### DISCUSSION TITLE
Should we migrate from graphql-import-node to @graphql-tools/graphql-file-loader

### DISCUSSION THREAD

**Discussion by the-ult** [Urigo/graphql-modules#2133] [Q&A]
We are using `.graphql` files to define our schema. And to be able to use them we use `graphql-import-node`
> If you wish to write your GraphQL SDL in a .graphql file, you need to make sure you are able to load it correctly in a NodeJS environment.
> To do so, use [graphql-import-node](https://github.com/ardatan/graphql-import-node) to make this process easier:

https://www.graphql-modules.com/docs/essentials/type-definitions#storing-sdl-in-graphql-files

However, this package does not seem to be actively maintained and it is recommended to use `@graphql-tools`

https://www.graphql-tools.com/docs/migration/migration-from-import

Should we use the `@grapql-tools` way? 
And how do you use this in combination with Apollo Server?

(should the docs be updated?)

**Comment by ardatan** [comment]
`graphql-import` is different than `graphql-import-node` :) So you can keep using graphql-import-node

**Reply by the-ult** [reply]
aah.. sorry.. my bad
